Sources/SQLKit/Query/SQLDataType.swift

```diff
@@ -6,34 +6,28 @@ public enum SQLDataType: SQLExpression {
     case text
     case real
     case blob
+    case custom(any SQLExpression)
```

Comment:
How breaking is this

Reply:
It isn't, it's just moved from further down in the file.